toonz/sources/translations/german/colorfx.ts

```diff
@@ -0,0 +1,723 @@
+    </message>
+    <message>
+        <source>Vert Offset</source>
+        <translation>Vertikaler Offset</translation>
+    </message>
+    <message>
+        <source>Noise</source>
+        <translation>Rauschen</translation>
+    </message>
+</context>
+<context>
+    <name>MovingSolidColor</name>
+    <message>
+        <source>Offset</source>
+        <translation>Offset</translation>
+    </message>
+    <message>
+        <source>Horiz Offset</source>
+        <translation>Horizontaler Offset</translation>
+    </message>
+    <message>
+        <source>Vert Offset</source>
+        <translation>Vertikaler Offset</translation>
+    </message>
+</context>
+<context>
+    <name>OutlineViewerStyle</name>
+    <message>
+        <source>OutlineViewer(OnlyDebug)</source>
+        <translation>Umriss-Betrachter (Nur Debug)</translation>
+    </message>
+    <message>
+        <source>Control Point</source>
+        <translation>Kontrollpunkt</translation>
+    </message>
+    <message>
+        <source>Center Line</source>
+        <translation>Mittellinie</translation>
+    </message>
+    <message>
+        <source>Outline Mode</source>
+        <translation>Umriss-Modus</translation>
+    </message>
+    <message>
+        <source>Distance</source>
+        <translation>Distanz</translation>
+    </message>
+    <message>
+        <source>distance</source>
+        <translation>Abstand</translation>
+    </message>
+</context>
+<context>
+    <name>ShadowStyle</name>
+    <message>
+        <source>Hatched Shading</source>
+        <translation>Schraffur</translation>
+    </message>
+    <message>
+        <source>Angle</source>
+        <translation>Winkel</translation>
+    </message>
+    <message>
+        <source>Density</source>
+        <translation>Dichte</translation>
+    </message>
+    <message>
+        <source>Length</source>
+        <translation>Länge</translation>
+    </message>
+</context>
+<context>
+    <name>ShadowStyle2</name>
+    <message>
+        <source>Plain Shadow</source>
+        <translation>Schatten (Standard)</translation>
+    </message>
+    <message>
+        <source>Angle</source>
+        <translation>Winkel</translation>
+    </message>
+    <message>
+        <source>Size</source>
+        <translation>Größe</translation>
+    </message>
+</context>
+<context>
+    <name>TAirbrushRasterStyle</name>
+    <message>
+        <source>Airbrush</source>
+        <translation>Airbrush</translation>
+    </message>
+    <message>
+        <source>Blur value</source>
+        <translation>Schattierungsgrad</translation>
+    </message>
+</context>
+<context>
+    <name>TBiColorStrokeStyle</name>
+    <message>
+        <source>Shade</source>
+        <translation>Schattierung</translation>
+    </message>
+</context>
+<context>
+    <name>TBlendRasterStyle</name>
+    <message>
+        <source>Blend</source>
+        <translation>Mengung</translation>
```

Comment:
Mengung is rarely used. Use Mischung or Mischen (preferred in this case) instead